common/backoff/cron.go

```diff
@@ -63,13 +69,24 @@ func GetBackoffForNextSchedule(cronSchedule string, startTime time.Time, closeTi
 	}
 	backoffInterval := nextScheduleTime.Sub(closeUTCTime)
 	roundedInterval := time.Second * time.Duration(math.Ceil(backoffInterval.Seconds()))
-	return roundedInterval
+
+	var jitter time.Duration
+	if jitterStartSeconds > 0 {
```

Comment:
Could be good to validate jitterStartSeconds somewhere to make sure its max value is lower than the time till next schedule

Reply:
Similarly, we should return an error when it's negative.